tests/e2e/logging/test_openshift-logging.py

```diff
@@ -171,3 +171,54 @@ def test_create_new_project_to_verify_logging(self, create_pvc_and_deploymentcon
         pod_obj.run_io(storage_type='fs', size=6000)
 
         self.validate_project_exists(pvc_obj)
+
+    @pytest.mark.polarion_id("OCS-651")
+    @tier4
+    def test_respin_elasticsearch_pod(self, create_pvc_and_deploymentconfig_pod):
+        """
+        Test to verify respin of elasticsearch pod has no functional impact
+        on logging backed by OCS.
+        """
+        pod_list = get_all_pods(namespace=constants.OPENSHIFT_LOGGING_NAMESPACE)
+        es_pod_list = [
+            pod.name for pod in pod_list if pod.name.startswith('elasticsearch')
+        ]
+
+        elasticsearch_pod_obj = get_pod_obj(
+            name=es_pod_list[1], namespace=constants.OPENSHIFT_LOGGING_NAMESPACE
+        )
+
+        # Respin the elastic-serach pod
+        elasticsearch_pod_obj.delete(force=True)
+
+        # Checks .operations index
+        es_pod_obj = get_pod_obj(
+            name=es_pod_list[2], namespace=constants.OPENSHIFT_LOGGING_NAMESPACE
+        )
+        operations_index = es_pod_obj.exec_cmd_on_pod(
+            command='es_util --query=.operations.*/_search?pretty', out_yaml_format=True
+        )
+        if operations_index:
+            logger.info(f"Successfully able to access the logs {operations_index}")
+        else:
+            logger.warn("Unable to access the logs from other ES pods")
+
+        # Gets the pod_list after the respin of elastic-search pods
+        pod_list = get_all_pods(constants.OPENSHIFT_LOGGING_NAMESPACE)
+        es_pod_list_after_respin = [
+            pod.name for pod in pod_list if pod.name.startswith('elasticsearch')
+        ]
+
+        # Checks the count of elastic search pods after the respin
```

Comment:
I don't think the 'if' statement is required as `check_health_of_clusterlogging` checks the complete logging's health anyways.

Reply:
Removed that check